src/msg/simple/Accepter.cc

```diff
@@ -313,7 +324,21 @@ void *Accepter::entry()
       }
       ldout(msgr->cct,1) << __func__ << " poll got error"  
  			  << " errno " << errno << " " << cpp_strerror(errno) << dendl;
-      break;
+ 			  
+ 	  if (msgr->cct->get_module_type() == CEPH_ENTITY_TYPE_OSD && hb){
+ 	    if (!unhealthy){
+	      ldout(msgr->cct,0) << "pool got error"
+	      		<< " reset heartbeat timeout"<< dendl;
+          msgr->cct->get_heartbeat_map()->reset_timeout(hb,0,0);
+          unhealthy = true;
+        }
+        else{
+         sleep(1);
```

Comment:
Hrm, I'm not sure about this bit. Is the idea that we've already marked ourselves as unhealthy, so just wait and hope the issue goes away?

Reply:
Yes, just want the accepter entry continue to  work before other osds put the failed osd out of the cluster.